sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/agents/package-info.java

```diff
@@ -1,13 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-/**
- * Package containing the clients and builders for Azure AI Search Knowledge Agents.
- * <p>
- * This package provides the main entry points for interacting with Azure AI Search Knowledge Agents, including both synchronous and asynchronous clients, as well as fluent builders for client configuration. The clients enable advanced knowledge retrieval and aggregation scenarios across multiple data sources configured in your Azure AI Search instance.
- * </p>
- * <p>
- * For more information about Azure AI Search Knowledge Agents, supported features, and usage guidelines, see the Azure documentation and the Azure SDK for Java guidelines.
- * </p>
- */
-package com.azure.search.documents.agents;
```

Comment:
Copy this into `com/azure/search/documents/knowledgebases` and change the `package` to `com.azure.search.documents.knowledgebases`